File: python/976_三角形的最大周长_easy.py
class Solution:
    def largestPerimeter(self, A: List[int]) -> int:
        A.sort(reverse=True)
        for i in range(0, len(A)-2):
            if A[i]<A[i+1]+A[i+2]:
                return A[i]+A[i+1]+A[i+2]
        return 0

File: cpp/976_三角形的最大周长_easy.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& A) {
        sort(A.begin(), A.end(), [](const int &a, const int &b){
            return a>b;
        });
        for(int i=0; i<A.size()-2; i++)
            if(A[i]<A[i+1]+A[i+2])
                return A[i]+A[i+1]+A[i+2];
        return 0;
    }
};